clients/java/src/main/java/io/camunda/zeebe/client/impl/command/RemovePermissionsCommandImpl.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.impl.command;
+
+import io.camunda.zeebe.client.api.JsonMapper;
+import io.camunda.zeebe.client.api.ZeebeFuture;
+import io.camunda.zeebe.client.api.command.FinalCommandStep;
+import io.camunda.zeebe.client.api.command.RemovePermissionsCommandStep1;
+import io.camunda.zeebe.client.api.command.RemovePermissionsCommandStep1.RemovePermissionsCommandStep2;
+import io.camunda.zeebe.client.api.command.RemovePermissionsCommandStep1.RemovePermissionsCommandStep3;
+import io.camunda.zeebe.client.api.command.RemovePermissionsCommandStep1.RemovePermissionsCommandStep4;
+import io.camunda.zeebe.client.api.response.RemovePermissionsResponse;
+import io.camunda.zeebe.client.impl.http.HttpClient;
+import io.camunda.zeebe.client.impl.http.HttpZeebeFuture;
+import io.camunda.zeebe.client.protocol.rest.AuthorizationPatchRequest.ActionEnum;
+import io.camunda.zeebe.client.protocol.rest.PermissionTypeEnum;
+import io.camunda.zeebe.client.protocol.rest.ResourceTypeEnum;
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.apache.hc.client5.http.config.RequestConfig;
+
+public class RemovePermissionsCommandImpl
+    implements RemovePermissionsCommandStep1,
+        RemovePermissionsCommandStep2,
+        RemovePermissionsCommandStep3,
+        RemovePermissionsCommandStep4 {
+
+  private final PatchAuthorizationCommand delegate;
+  private final HttpClient httpClient;
+  private final JsonMapper jsonMapper;
+  private final String path;
+  private final RequestConfig.Builder httpRequestConfig;
+
+  public RemovePermissionsCommandImpl(
+      final long ownerKey, final HttpClient httpClient, final JsonMapper jsonMapper) {
+    this.httpClient = httpClient;
+    this.jsonMapper = jsonMapper;
+    path = "/authorizations/" + ownerKey;
+    httpRequestConfig = httpClient.newRequestConfig();
+    delegate = new PatchAuthorizationCommand(ActionEnum.REMOVE);
```

Comment:
❌ The default request timeout should be set in the constructor as well. Please have a look at other commands, like SetVariables or DeployResources